src/Components/Aspire.OpenAI/MEAIPackageOverrides.targets

```diff
@@ -0,0 +1,13 @@
+<Project>
+  <ItemGroup>
+    <!--
+      Microsoft.Extensions.AI depends on 9.x packages, even on net8.0, so we have to override central package management
+      to avoid "package downgrade" build errors. This is only used when referencing Aspire.OpenAI and doesn't break
+      compatibility with net8.0.
+    -->
+    <PackageReference Include="Microsoft.Extensions.Primitives" VersionOverride="9.0.0-*" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" VersionOverride="9.0.0-*" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" VersionOverride="9.0.0-*" />
+    <PackageReference Include="System.Text.Json" VersionOverride="9.0.0-*" />
```

Comment:
[This PR](https://github.com/dotnet/aspire/pull/6194) should simplify that, right? Or do we also want to use the package in apps targeting 8.0?

Reply:
> Or do we also want to use the package in apps targeting 8.0?

Yes. We have no choice, because M.E.AI.Abstractions targets the 9.x versions of these packages (even on `net8.0`).

Reply:
We shouldn't be using wildcards `*`s pacakge versions.

As for using STJ 9 on net8.0, I think that is something to think hard about in MEAI before shipping, since 9 is STS and 8 is LTS.

Reply:
@SteveMCarroll

Reply:
> The current status for this is that MEAI.Abstractions will depend on STJ v8, whereas MEAI will depend on STJ v9, right?

When targeting < net9.0, yes.

> If so then the Aspire integration package will still have to depend on STJv9 (even on net8.0)... Is this going to be a problem?

Not sure. Who do we think will be concerned?

Reply:
@eerhardt Your comment above gave me the impression this was a concern. Can you let me know if it is?

Reply:
Generally there are 2 concerns with lifting libraries out of the shared framework:

1. (technical) It is another .dll that needs to be distributed with your application. Also, it is no longer ReadyToRun'd because we aren't using the assembly in the shared framework (which is RID-specific and R2R'd).
2. (policy) the servicing lifetime of System.Text.Json 9.0 ends before the servicing lifetime of System.Text.Json 8.0. If you are staying on `net8.0` TFM, you probably want to keep on a long-term support framework. But using MEAI (not .Abstractions) means that you are signing up for being on the short-term support System.Text.Json. Meaning you will need to move up to the v10 System.Text.Json faster than you would have if you stayed on the v8.

Reply:
Thanks @eerhardt. Based on this I'm going to complete the PR using the API design [here](https://github.com/dotnet/aspire/pull/6225#issuecomment-2470704222).

If anyone doesn't think this is the best API shape then please speak up asap!